modules/lindyn/src/LinDyn.f90

```diff
@@ -251,6 +251,7 @@ subroutine StateMatrices(MM, CC, KK, AA, BB, errStat, errMsg)
    integer(IntKi)                          :: errStat2    ! Status of error message
```

Comment:
The typo was also in the comment above (the minus should be a plus in B).... 
```
!! A = [   0        I       ]     B = [0      ]
!!     [-M^{-1}K   -M^{-1}C ]       = [ + M^{-1}]  
```

Reply:
Thanks for the review. I've fixed that comment.